api/internal/handler/consumer/consumer.go

```diff
@@ -101,6 +102,10 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 	}
 	input.ID = input.Username
 
+	if err := utils.SchemaCheck("main.consumer", input); err != nil {
```

Comment:
`GenericStore` has already implement it, plz have a look at `validate.go` and `validate_test.go`, We can init it with `GenericStore` at same time.

Reply:
done.